install/linux.md

```diff
@@ -75,41 +75,24 @@ Lizenz und enthalten Tracking Software. Deswegen nutzen wir VSCodium.
 
 Startet VS Code nach der Installation, klickt in der linken Leiste auf `Extensions` (Unterstes Symbol) und sucht nach „LaTeX Language Support“, installiert die Extension mit einem Klick auf den `install` Knopf.
 
-### Anaconda
-
-__Wichtig__: Wir wollen Python 3.9 für Linux. [64-Bit (x86) Installer]
-
-- [Anaconda](https://www.anaconda.com/products/individual#Downloads): Python und Bibliotheken
-
-Im Terminal im Ordner mit der heruntergeladenen Datei den Befehl:
-
-```
-$ bash Anaconda3-2022.10-Linux-x86_64.sh -p ~/.local/anaconda3 -b
-```
-
-ausführen. Wenn es eine neue Version von Anaconda gibt, ändert sich der Dateiname und muss entsprechend angepasst werden.
-
-Öffnet die Datei `~/.bashrc` und fügt diese beiden Zeilen am Ende hinzu:
-
+### Mambaforge
+
+Hier müssen im Terminal die folgenden Zeilen eingegeben werden:
```

Comment:
Wollen wir für interessierte mehr Erklärungen bieten? Also z.B.
Es gibt verschiedene Wege, _python_ zu installieren. Wir zeigen euch hier, wie ihr mit [Mamba](https://mamba.readthedocs.io/en/latest/index.html) _python_ und die Pakete, die wir vorstellen, installieren könnt. Dazu müssen im Terminal die ...

Reply:
Weiß nicht. Wäre vielleicht etwas für die "Fortgeschrittenen Materialien". 
Hier verstehen die Leute im Zweifel eh noch nicht, "was" und "warum" sie es tun.
Ich glaube an dieser Stelle fehlt da etwas der Mehrwert. Nach dem Kurs ist das vielleicht interessanter.

Reply:
Das muss in die intro folien. 

In der Anleitung würde ich erstmal blind folgen lassen.

Ich setze mich am Wochenende an einen ersten Update Versuch für die Intro Folien

Reply:
Ich würde zumindest erwähnen, dass es die Installation für python ist.

Reply:
Ahja, das ist richtig. Statt "Mambaforge" könnte man "Python Installation: Mambaforge" schreiben